test/Dialect/FIRRTL/errors.mlir

```diff
@@ -191,7 +191,7 @@ firrtl.circuit "Foo" {
 firrtl.circuit "Foo" {
   firrtl.module @Foo() {
   // expected-error @+1 {{'firrtl.mem' op attribute 'writeLatency' failed to satisfy constraint: 32-bit signless integer attribute whose minimum value is 1}}
-    %m = firrtl.mem Undefined {depth = 32 : i64, name = "m", readLatency = 0 : i32, writeLatency = 0 : i32} : !firrtl.bundle<>
+    %m = firrtl.mem Undefined {depth = 32 : i64, name = "m", portNames = ["rw"], readLatency = 0 : i32, writeLatency = 0 : i32} : !firrtl.bundle<>
```

Comment:
I don't know what this is, but why does it need to be added?

Reply:
Ah, I meant to write a comment about this inline in the PR diff, but forgot. Here it looks like the `firrtl.mem` op requires the `portNames` attribute to be set, but before the LLVM submodule bump, it would hit the `writeLatency` error before getting to the `portNames` error. I don't think this has anything to do with the rest of the changes in this PR, and something about upstream MLIR changed the order in which the attributes were being verified.

The original test intent clearly seems to be about the `writeLatency` value, and I think it was just an oversight that the `portNames` attribute was omitted.